platform/features/table/src/controllers/RTTelemetryTableController.js

```diff
@@ -69,53 +69,36 @@ define(
         RTTelemetryTableController.prototype = Object.create(TableController.prototype);
 
         /**
-         Override the subscribe function defined on the parent controller in
-         order to handle realtime telemetry instead of historical.
+         *
          */
-        RTTelemetryTableController.prototype.subscribe = function () {
-            var self = this;
-            self.$scope.rows = undefined;
-            (this.subscriptions || []).forEach(function (unsubscribe){
-                unsubscribe();
-            });
+        RTTelemetryTableController.prototype.addHistoricalData = function () {
```

Comment:
Better to have the TableController superclass implement a no-op then requiring a no-op to be implemented in the subclass when the functionality is optional.